quil-py/quil/instructions/__init__.pyi

```diff
@@ -1175,6 +1189,303 @@ class FrameIdentifier:
         If any part of the instruction can't be converted to valid Quil, it will be printed in a human-readable debug format.
         """
 
+@final
+class CallArgument:
+    """An argument to a `Call` instruction.
+
+    This may be expressed as an identifier, a memory reference, or an immediate value. Memory references and identifiers require a corresponding memory region declaration by the time of
+    compilation (at the time of call argument resolution and memory graph construction to be more precise).
+
+    Additionally, an argument's resolved type must match the expected type of the corresponding `ExternParameter`
+    in the `Externsignature`.
+    """
```

Comment:
```suggestion
    """An argument to a `Call` instruction.

    This may be expressed as an identifier, a memory reference, or an immediate value. Memory references and identifiers require a corresponding memory region declaration by the time of
    compilation (at the time of call argument resolution and memory graph construction, to be more precise).

    Additionally, an argument's resolved type must match the expected type of the corresponding `ExternParameter`
    in the `ExternSignature`.
    """
```